src/pages/jonghyuk/Main/Comment.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useState } from 'react';
+import Newcomment from './Newcomment';
+
+const Comment = () => {
+  const [comment, setComment] = useState('');
+  const hendleComment = event => {
+    setComment(event.target.value);
+  };
+  const [commentArray, setCommentArray] = useState([]);
+  const eventComment = () => {
+    const result = [...commentArray, comment];
+    setCommentArray(result);
+    setComment('');
+  };
+
+  const changeColor = comment.length > 0;
```

Comment:
boolean값을 뜻하는 변수명은 is, has 라는 prefix 를 붙혀주는게 좋습니다 
(보자마자 아! 이건 boolean값을 가지는 변수명이구나 라는 것을 catch 하기 위해)

how about
```suggestion
  const isCommentFilled = comment.length > 0;
```

Reply:
넵 감사합니다 참고하도록 하겠습니다!